catalog/src/test/java/it/eng/catalog/ApplicationCatalogTest.java

```diff
@@ -0,0 +1,14 @@
+package it.eng.catalog;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
```

Comment:
Leftover from integration testing investigation?

Reply:
Removed it, since I found a way to test without it